### PR TITLE
Add Footer to User Page

### LINKED ISSUE #1016: User Profile pages don't show common page footer
**Description**
I noticed that the common page footer is not present on user profile pages. Is that intentional?

**Steps to Reproduce**
1. Nav to any user profile page e.g. https://stacker.news/WeAreAllSatoshi
2. Observe that the page footer is not present

**Expected behavior**
Expect page footer on profile pages

**Screenshots**
image

**Logs**
N/A

**Environment:**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by itsrealfake**
this closes #1016

## Description

Shows a `Footer` on the User Page when the user has a bio but shows no `Footer` when user bio is absent.

## Screenshots

Skinny screen:
image

Thick screen:
image

Hidden when there's no bio:
image

## Checklist

**Are your changes backwards compatible? Please answer below:**

yes

yes

**For frontend changes: Tested on mobile? Please answer below:**

Tested with mobile emulation in browser

### CODE REVIEW

**Comment by itsrealfake** [comment]
this branch should have been `1016-...` my eyeball copy-pasta game is weak af.

File: pages/[name]/index.js [hunk 1]
```diff
@@ -122,6 +123,7 @@ export default function User ({ ssrData }) {
                     </div>
                 )}
           </div>)}
+      {user.bio && <Footer />}
```
[the remark this answers is not in the document]
Comment (itsrealfake) [reply pages/[name]/index.js]: used `user?.bio` instead

**Comment by benalleng** [comment]
I think that it still doesn't look great when the user's profile is sparse, especially in the mobile view you showed. while their bio isn't empty it certainly feels like the footer takes up way too much space, even though you did scroll down to it.

I will again suggest that we try the footer like `/404` as it is kind of a footer-lite and isn't quite as obtrusive on pages like this that may frequently be empty or sparse

**Comment by itsrealfake** [comment]
thanks for the feedback & resolution @benalleng + @huumn

**Pull request merged**